src/python/pants/init/plugin_resolver.py

```diff
@@ -85,6 +85,7 @@ async def resolve_plugins(
             interpreter_constraints=request.interpreter_constraints or InterpreterConstraints(),
             description=f"Resolving plugins: {', '.join(req_strings)}",
         ),
+        **implicitly(),
```

Comment:
Errors seem to be of the form:

```
TypeError: build_pex() got an unexpected keyword argument '__implicitly'
```

🤯

Reply:
`from pex.bin.pex import build_pex`

Wrong location for the import ... ho hum ...